CHANGES

```diff
@@ -43,6 +43,7 @@
 - MAST: Support downloading data from multiple missions from the cloud [#1275]
 - JPLHorizons: Fix queries for major solar system bodies when sub-observer or sub-solar positions are requested. [#1268]
 - JPLHorizons: Fix bug with airmass column. [#1284]
+- docs: Clarifying install instructions [#1270]
```

Comment:
no changelog is needed for docs changes, but will remove it later in the actual release.

Reply:
Good to know. Thanks.